plugins/outputs/cloudwatch/cloudwatch_test.go

```diff
@@ -96,7 +96,8 @@ func TestBuildMetricDatums(t *testing.T) {
 	}
 
 	invalidDistribution := distribution.NewDistribution()
-	invalidDistribution.AddEntry(-1, 1)
+	err := invalidDistribution.AddEntry(-1, 1)
+	assert.Error(err)
```

Comment:
Instead of only `assert.Error`, using `assert.EqualErrorf(t, err, expectedErrorMsg, "Error should be: %v, got: %v", expectedErrorMsg, err)` would be a better idea to confirm if the metrics are actually have that kind of error: . In this case, the negative value.

Reply:
Good idea, will do.